DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -531,7 +531,7 @@ public void RegisterCentreManager_calls_all_relevant_registration_methods()
                 () =>
                     passwordDataService.SetPasswordByCandidateNumber(A<string>._, A<string>._)
             ).MustHaveHappenedOnceExactly();
-            A.CallTo(() => registrationDataService.RegisterAdmin(centreManagerModel, userId))
+            A.CallTo(() => registrationDataService.RegisterAdmin(A<AdminAccountRegistrationModel>._))
                 .MustHaveHappenedOnceExactly();

```

Comment:
Should specify the model precisely here, as this is a "MustHaveHappenedOnce" case. Just to make sure we are passing in what we think we should be.